.github/workflows/main.yml

```diff
@@ -53,4 +53,32 @@ jobs:
       - run: |
           echo "I am protected!"
           sleep 5
+          
+  always_unlock_test_1:
```

Comment:
These are good tests!

Would this job show up as failed? I'm not sure how to best test this while still having the job pass. Perhaps we can run the check in a subsequent step?